src/pid-file/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "mz-pid-file"
```

Comment:
What's up with this file?

Reply:
ahh shoot it got pulled in from a rebase, it used to exist but was deleted recently. I'll fix this